sqlmesh/migrations/v0052_add_normalize_name_in_environment_naming_info.py

```diff
@@ -0,0 +1,27 @@
+"""Add flag that controls whether environment names will be normalized."""
+
+from sqlglot import exp
+
+
+def migrate(state_sync, **kwargs):  # type: ignore
+    engine_adapter = state_sync.engine_adapter
+    environments_table = "_environments"
+    if state_sync.schema:
+        environments_table = f"{state_sync.schema}.{environments_table}"
+
+    alter_table_exp = exp.AlterTable(
```

Comment:
in the future, we should make a builder for alter table, low pri

Reply:
yep 👍